Tools/measure-macro-performance.sh

```
@@ -10,6 +10,18 @@ BASELINE_FILE="Tools/macro-performance-baseline.json"
 THRESHOLD_PERCENT=20
 UPDATE_BASELINE=0
 IN_PROCESS=0
+EXPLICIT_REPORT_ONLY=0
+# Honor an externally-set `ENFORCE_REGRESSION_GATE` so callers can opt out of
+# the regression gate even inside GitHub Actions (e.g. a workflow that wants
+# the timing report without failing the run). Default to enforcing on
+# `GITHUB_ACTIONS=true` and to report-only otherwise.
+if [[ -n "${ENFORCE_REGRESSION_GATE:-}" ]]; then
+  ENFORCE_REGRESSION_GATE="$ENFORCE_REGRESSION_GATE"
```

Shellcheck (0.11.0)

[info] This variable is assigned to itself, so the assignment does nothing.

(SC2269)